charts/kargo/templates/controller/configmap.yaml

```diff
@@ -23,7 +23,11 @@ data:
   {{- if .Values.kubeconfigSecrets.kargo }}
   KUBECONFIG: /etc/kargo/kubeconfigs/kubeconfig.yaml
   {{- end }}
-  GLOBAL_CREDENTIALS_NAMESPACES: {{ quote (join "," .Values.controller.globalCredentials.namespaces) }}
+  {{- if ( len .Values.controller.globalCredentials.namespaces ) }} 
+  SHARED_RESOURCES_NAMESPACE: {{ quote ( index .Values.controller.globalCredentials.namespaces 0 ) }}
```

Comment:
I think this should be removed now, as otherwise only the first entry on the list will be checked due to the code changes.